var/spack/repos/builtin/packages/flang/package.py

```diff
@@ -30,29 +30,19 @@
 class Flang(CMakePackage):
     """Flang is a Fortran compiler targeting LLVM."""
     homepage = "https://github.com/flang-compiler/flang"
-    url      = "https://github.com/flang-compiler/flang/archive/flang_20180319.tar.gz"
+    url      = "https://github.com/flang-compiler/flang/archive/flang_20180612.tar.gz"
 
     version('develop', git='https://github.com/flang-compiler/flang', branch='master')
-    version('20180319', 'e659bad83b791f90af2c5cd672864669')
```

Comment:
Don't drop old versions!

Reply:
That version does not exist anymore. We've since dropped that tag from our repository.